index.ejs

```diff
@@ -22,8 +22,8 @@
 
         <link type="text/css" rel="stylesheet" href="../api/files/style" />
         
-        <script src="vendor/jquery/3.2.0/dist/jquery.js"></script>
-        <script src="vendor/jquery-migrate/3.0.0/dist/jquery-migrate.min.js"></script>
+        <script src="vendor/jquery/3.6.0/dist/jquery.js"></script>
+        <script src="vendor/jquery-migrate/3.4.0/dist/jquery-migrate.min.js"></script>
 
         <script src="vendor/jquery-ui/1.12.1/ui/widget.js"></script>
         <script src="vendor/jquery-ui/1.12.1/ui/widgets/mouse.js"></script>
```

Comment:
We should also upgrade `jquery-ui` to at least version `1.13.0` https://jira.dhis2.org/browse/DHIS2-13510

Reply:
(If you want to do that in a separate PR that's fine too)

Reply:
https://jqueryui.com/upgrade-guide/1.13/